siege_utilities/reporting/analytics/polling_analyzer.py

```
@@ -371,107 +238,108 @@ def create_polling_summary(self,
                        ax=ax,
-                       cbar_kws={'label': 'Sessions'})
-            
+                       cbar_kws={'label': metric.title()})
+
             ax.set_title(title, fontsize=14, fontweight='bold')
             ax.set_xlabel('')
             ax.set_ylabel('')
-            
+
             plt.tight_layout()
             return fig
```

Ruff (0.15.10)

[warning] Consider moving this statement to an `else` block

(TRY300)